collections/Hamilton/MS_Hamilton_15.xml

```diff
@@ -49,37 +47,267 @@
+                     
+                     
+                    
+                     
+                     <note>The sequence of material on fols. 160va-188va is closely paralleled in <bibl type="ms" subtype="external">Gotha, Forschungsbibliothek, Chart. A. 1928</bibl> fols. 180ra-217ra, in a collection of Lent sermons there attributed to Henricus de Frimaria.</note>
+                     <note>Attributed to <persName role="aut" key="person_4472713" cert="low">Simon Cassia</persName> by early owners: fol. 249r 'pars yemalis sucus symonis'; fol. i recto, 'Iste liber intitulatur pars yemalis Sermonum Symonis de Cassia'; contents label, front cover.</note>
+                  </msItem>
+                  
+                  
+                  <msItem>
+                     <locus>(fol. 239r; 239v blank)</locus>
+                     <incipit><quote>Mortuo non prohibeas graciam</quote> Ecci. 7o. Tanto graciosius est opus misercordie subuenientis. Et quia ne<supplied reason="omitted">m</supplied>o pauperior hiis qui in purgatorio detinentur </incipit>
+                     <explicit defective="true">Unde sensus est Non exibunt</explicit>
+                     <note>Schneyer VI, p. 431 (no. 42; the full incipit is 'Quanto major est necessitas indigentis, tanto...'); also in <bibl type="ms" subtype="external">Zwettl, Zisterzienserstift, Cod. 206</bibl>. Ends unfinished.</note>
+                  </msItem>
+                  <msItem>
+                     <locus>(fols. 240ra-244vb)</locus>
+                     <author key="person_24629057"><supplied>Johannes Andreae</supplied></author>
+                     <title key="work_6241">De modo observandi interdictum</title>
+                     <incipit>De modo obseruandi interdictum memoriale quoddam simplicibus colligendo</incipit>
+                     <explicit defective="true">non diffiniendo sed propo</explicit>
+                     <note>Ends imperfect (at f. 11r of ISTC ia00629000).</note>
+                     <!-- https://app.digitale-sammlungen.de/bookshelf/bsb00044163/view?currentUrl=%2Fview%2Fbsb00044163%3Fview%3DImageView&manifest=https%3A%2F%2Fapi.digitale-sammlungen.de%2Fiiif%2Fpresentation%2Fv2%2Fbsb00044163%2Fmanifest&canvas=https%3A%2F%2Fapi.digitale-sammlungen.de%2Fiiif%2Fpresentation%2Fv2%2Fbsb00044163%2Fcanvas%2F15&view=ImageView -->
+                  </msItem>
+                  <msItem>
+                     <locus>(fols. 245ra-246va; 246vb, 247r blank)</locus>
+                     <title type="desc" key="work_15091">Treatise on the mass</title>
+                     <incipit defective="true">in f<hi rend="superscript">r</hi>mis. Nam de miraculo veteris panis exodi</incipit>
+                     <explicit>omnem suam dulcedinem infundit in epulum quod nobis prestare dignetur</explicit>
```

Comment:
Probably a variant of Conrad de Brundelsheim, sermon no. 124.

Reply:
Good thought but it doesn't seem to be : 1484 edition http://www.ub.uni-koeln.de/cdm/compoundobject/collection/inkunabeln/id/109982/rec/1 around image 436